flexinput/src/main/res/values/styles.xml

```diff
@@ -144,8 +144,11 @@
 
     <style name="FlexInput.AddContent.FloatingActionButton.Launcher" parent="FlexInput.AddContent.Base.FloatingActionButton">
```

Comment:
probably shouldn't be under `FloactingActionButton` anymore?

Maybe rename path to just `FlexInput.AddContent.Base.Button`:
```
    <style name="FlexInput.AddContent.Button" parent="FlexInput.AddContent">
        <item name="android:layout_width">wrap_content</item>
        <item name="android:layout_height">wrap_content</item>
        <item name="android:elevation" tools:targetApi="lollipop">8dp</item>
        <item name="android:tint">@android:color/white</item>
        <item name="android:tintMode" tools:targetApi="lollipop">src_atop</item>
    </style>
    <style name="FlexInput.AddContent.Button.Send">
        <item name="android:src">@drawable/ic_send_24dp</item>
        <item name="android:layout_marginRight">16dp</item>
        <item name="layout_anchorGravity">top|right|end</item>
    </style>

    <style name="FlexInput.AddContent.Button.Launcher">
        <item name="android:src">@drawable/ic_launch_24dp</item>
        <item name="android:layout_gravity">top|start|left</item>
        <item name="layout_anchorGravity">top|start|left</item>
        <item name="background">?selectableItemBackgroundBorderless</item>
    </style>
```